runtime/doc/builtin.txt

```diff
@@ -3786,6 +3787,16 @@ getbufvar({buf}, {varname} [, {def}])				*getbufvar()*
 		Return type: any, depending on {varname}
 
 
+getcellpixels()						*getcellpixels()*
+		Returns a |List| of cell pixel size.
+		List format is [xpixels, ypixels].
```

Comment:
Only works on Unix. For gVim, returns `[5, 10]`, on other systems, returns `[-1, -1]`.

Reply:
Shouldn't gVIm return [-1,-1] until gVim gets a proper implementation?